code42.json

```diff
@@ -1924,6 +1924,964 @@
+                },
+                {
+                    "data_path": "action_result.data",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "run query",
+            "identifier": "run_query",
+            "description": "Search for Code42 file events",
+            "verbose": "You can use wildcards (*,?) with most string-based fields. All query parameters are in logical AND with each other.",
+            "type": "investigate",
+            "read_only": true,
+            "parameters": {
+                "start_date": {
+                    "description": "Beginning of date range to search. Time (in UTC) may also be supplied but is not required.",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "default": "",
+                    "order": 0
+                },
+                "end_date": {
+                    "description": "End of date range to search. Time (in UTC) may also be supplied but is not required.",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "default": "",
+                    "order": 1
+                },
+                "file_hash": {
+                    "description": "Md5 or sha256 hash of a file",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": true,
+                    "contains": [
+                        "md5",
+                        "sha256"
+                    ],
+                    "default": "",
+                    "order": 2
+                },
+                "file_name": {
+                    "description": "File name",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": true,
+                    "default": "",
+                    "order": 3
+                },
+                "file_path": {
+                    "description": "File path",
+                    "data_type": "string",
+                    "order": 4,
+                    "primary": true,
+                    "contains": [
+                        "file path"
+                    ]
+                },
+                "file_category": {
+                    "description": "File category",
```

Comment:
Could make these description slightly better by becoming inspired from the py42 docs: https://py42docs.code42.com/en/stable/methoddocs/filleeventqueries.html#py42.sdk.queries.fileevents.filters.file_filter.FileCategory

^ In this case, something like `Filter events by category of the file observed`.

Reply:
I pulled this pattern from the existing app but have no problem with making them a little more detailed

Reply:
Done!